src/components/settings/SettingsLMSTab/ErrorReporting/DownloadCsvButton.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useEffect } from 'react';
+import { saveAs } from 'file-saver';
+import PropTypes from 'prop-types';
+import {
+  Toast, StatefulButton, Icon, Spinner, useToggle,
+} from '@edx/paragon';
+import { Download, Check } from '@edx/paragon/icons';
+import { isStrictlyArray } from './utils';
+
+const DownloadCsvButton = ({ data }) => {
+  const [buttonState, setButtonState] = useState('pageLoading');
+
+  useEffect(() => {
+    if (data && data.length) {
+      setButtonState('default');
+    }
+  }, [data]);
+
+  const getCsvFileName = () => {
```

Comment:
is it worth to include channel code/config name/some other identifier in the CSV name? I'm not completely sure how I feel one way or the other, just a thought

Reply:
this is good feedback - i'd say for V1 lets not worry about additional columns. we are eventually going to power this by the backend and add a bunch more columns that dont appear in the view and we may add this stuff then.